lib/services/users.pm

```diff
@@ -147,6 +147,8 @@ sub full_users_check {
         record_info('Unsupported on non-gnome', "This test is only supported on gnome, quit for your DESKTOP is $desktop", result => 'fail');
         return;
     }
+    # reset consoles before select x11 console will make the connect operation more stable
+    reset_consoles;
```

Comment:
Should only needed on s390?

Reply:
reset_consoles work for all platform while must for s390x, so I updated code here and waiting for the verification log.
http://openqa.nue.suse.com/tests/6129942#live